controllers/resources_test.go

```diff
@@ -268,4 +280,165 @@ var _ = Describe("Resource creation and cleanup tests", func() {
 		}
 	})
 
+	Context("ServiceMonitor Creation test", func() {
```

Comment:
Can change the name of the `Context` description because we are not only doing creation tests.